### DISCUSSION TITLE
Welcome to domino Discussions!

### DISCUSSION THREAD

**Discussion by luiztauffer** [Tauffer-Consulting/domino#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manuganji** [comment]
Hey, thanks for making this repository. I have worked for the last few months on a similar tool. But I appreciate that you are already quite ahead. I'll explore more and reach out to you with ways to contribute.

**Reply by vinicvaz** [reply]
Hey @manuganji , nice to have you here!
Feel free to contact us with any questions or feedback!
Just to know, what tool you have been working with? 
Thanks!
